File: FitNessCompanionApp/FitNessCompanionApp/Model/Order.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace FitNessCompanionApp.Model
{
    class Order
    {
        public Order(string OrderDate, User User, List<OrderItem> OrderedContent, double TotalPrice)
        {
            this.OrderDate = OrderDate;
            this.User = User;
            this.OrderContent = OrderedContent;
            this.TotalPrice = TotalPrice;
        }

        #region Fields
        [JsonProperty("orderDate")]
        public string OrderDate { get; set; }

        [JsonProperty("user")]
        public User User { get; set; }

        [JsonProperty("orderContent")]
        public List<OrderItem> OrderContent { get; set; }

        [JsonProperty("totalPrice")]
        public double TotalPrice { get; set; }
        #endregion
    }
}

File: Gym-Website/src/app/Model/Order.ts
import { OrderItem } from './OrderItem';
import { User } from './User';

export class Order
{
    orderDate:string;
    orderContent:OrderItem[];
    totalPrice:number;
    user:User;

    constructor(orderContent:OrderItem[], user: User, totalPrice:number)
    {
        this.orderContent = orderContent;
        this.orderDate = new Date().toLocaleString().split(',')[0];
        this.totalPrice = totalPrice;
        this.user = user;
    }
}